src/main.c

```diff
@@ -201,195 +208,224 @@ int main()
       RELAY = 1;
 
       // run every ~1 secs
-      if (count % 4 == 0) {
+      if ((count & 3) == 0) {
```

Comment:
so this is where Im curious which version of sdcc you are using. I'm using 3.5.0, and the assembly produced (looking at main.asm) already does this under the hood:
```
;	src/main.c:204: if (count % 4 == 0) {
	mov	a,_count
	anl	a,#0x03
	jz	00333$
	sjmp	00104$
00333$:
```

So I'm wondering if this is a readability thing?